.prettierignore

```diff
@@ -1,5 +1,5 @@
-src/generated/
-review/*.api.md
-*.yml
-*.yaml
-*.d.ts
+**/src/generated/
+**/review/*.api.md
+**/*.yml
+**/*.yaml
+**/*.d.ts
```

Comment:
Is this necessary?

Reply:
I'm also curious if this makes a difference

Reply:
`pretty-quick` applies the rules in `.prettierignore` file in a way different from `prettier`. The former applies the rules relative to the root of source control tree (see [here](https://github.com/azz/pretty-quick/blob/9290ae433655935f23513174a409ec537b407cc1/src/index.js#L47)) but the latter applies the rules relative to the current directory (see [here](https://github.com/prettier/prettier/blob/86ad2997685a9ac86617502531e9d203976d23ea/src/cli/util.js#L445)). It is confusing and there is an issue opened for it here: https://github.com/azz/pretty-quick/issues/95#issuecomment-567459834.